app.rb

```diff
@@ -43,6 +47,14 @@ def initialize
     cache_control :public, :must_revalidate, :max_age => 60
   end
 
+  get '/img/:file' do |file|
+    send_file File.join(cloudgov_path, 'img', file)
+  end
+
+  get '/font/:file' do |file|
+    send_file File.join(cloudgov_path, 'font', file)
+  end
```

Comment:
I'm not familiar with how Sinatra assets work. I tried adding those to assets (code below), which didn't work. It looks like the assets pipeline is for concatenating/minifying/etc, with the results being available through some special tags (like `<%= javascript_path 'app' %>` in `layout.erb for example).

```ruby
assets.append_path File.join(cloudgov_path, 'img')
assets.append_path File.join(cloudgov_path, 'font')
```